cherrypy/lib/cpstats.py

```diff
@@ -283,22 +288,27 @@ def readlines(self, sizehint=0):
         return lines
 
     def close(self):
+        """Close method for ByteCountWrapper."""
         self.rfile.close()
 
     def __iter__(self):
+        """Iterate method for ByteCountWrapper."""
         return self
 
     def next(self):
+        """Next method for ByteCountWrapper."""
         data = self.rfile.next()
         self.bytes_read += len(data)
         return data
 
 
 def average_uriset_time(s):
```

Comment:
Observation during exploratory work: the `cpstats` module can be configured with something like
```python
{
    '/': {
        'tools.cpstats.on': True,
        'tools.cpstats.uriset': 'ROOT',
    },
    '/cpstats': {
        'tools.cpstats.uriset': 'STAT',
    },
    '/something': {
        'tools.cpstats.uriset': 'SMTH',
    },
}
```

And that would tag different suburls with identifiers, showing stats per identifier.